app/simulation/utils.go

```diff
@@ -50,3 +51,21 @@ func simulateFromSeed(t *testing.T, app *regen.RegenApp, config simtypes.Config)
 		app.AppCodec(),
 	)
 }
+
+// removeICAFromSimulation is a utility function that removes from genesis exporting due to a panic bug.
+//
+// TODO: remove after https://github.com/cosmos/ibc-go/issues/2151 is resolved
+func removeICAFromSimulation(app *regen.RegenApp) {
+	remove := func(target string, mods []string) []string {
+		for i, mod := range mods {
+			if mod == target {
+				return append(mods[:i], mods[i+1:]...)
+			}
+		}
+		return mods
+	}
+
+	icaModName := ica.AppModule{}.Name()
+
+	app.ModuleManager.OrderExportGenesis = remove(icaModName, app.ModuleManager.OrderExportGenesis)
+}
```

Comment:
unfortunately ICA module does not implement `AppModuleSimulation` so the module does initialize with any state during simulations. This in turn causes `ExportGenesis` calls on the `regen.App` to panic due to a bug described here https://github.com/cosmos/ibc-go/issues/2151. 

This is a temporary workaround until the above issue is resolved on the IBC-go repo, or a more preferable solution is found.

Reply:
Work around for now works for me.